docs/app/Examples/views/Statistic/StatisticExamples.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
+import StatisticTypesExamples from './Types/StatisticTypesExamples';
+
+export default class StatisticExamples extends Component {
+  render() {
+    return (
+      <div>
+        <StatisticTypesExamples />
+      </div>
```

Comment:
A single basic example, just to prove the component.  Extended examples can be added on a separate PR.